src/Nethermind/Nethermind.Runner/Ethereum/Steps/DatabaseMigrations.cs

```diff
@@ -152,6 +152,12 @@ private long MinBlockNumber
 
         private void RunBloomMigration(CancellationToken token)
         {
+            BlockHeader GetMissingBLockHeader(long i)
+            {
+                if (_logger.IsWarn) _logger.Warn(GetLogMessage("warning", $"Header for block {i} not found. Database might be corrupted and BloomDb will not be correct."));
+                return new BlockHeader(Keccak.Zero, Keccak.Zero, Address.Zero, UInt256.Zero, 0L, 0L, UInt256.Zero, Bytes.Empty);
```

Comment:
Create static EmptyBlockHeader here (not in BlockHeader) to avoid new

Reply:
Doesn't have to be static though